### PR TITLE
Methods to create table on tableset

### PR DESCRIPTION
**Pull request by tdietrich513**
Enables more straightforward access to the backing table to manage it without having to spin up your own reference.

### CODE REVIEW

File: src/WindowsAzure/Table/ITableSet.cs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
```
Comment (dtretyakov) [inline src/WindowsAzure/Table/ITableSet.cs]: General concern in this interface don't use namespaces from Azure SDK library.

File: src/WindowsAzure/Table/ITableSet.cs [hunk 2]
```diff
@@ -165,5 +167,51 @@ public interface ITableSet<TEntity> : IOrderedQueryable<TEntity> where TEntity :
...
+
+        /// <summary>
+        ///    Creates the table if it does not already exist.
+        /// </summary>      
+        /// <param name="requestOptions">A <see cref="T:Microsoft.WindowsAzure.Storage.Table.TableRequestOptions"/> object that specifies additional options for the request.</param>
+        /// <param name="operationContext">An <see cref="T:Microsoft.WindowsAzure.Storage.OperationContext"/> object that represents the context for the current operation.</param>
+        /// <returns>
+        /// <c>true</c> if table was created; otherwise, <c>false</c>.
+        /// </returns>                  
+        void CreateIfNotExists(TableRequestOptions requestOptions = null, OperationContext operationContext = null );
```
Comment (dtretyakov) [inline src/WindowsAzure/Table/ITableSet.cs]: So, please remove this overload,

File: src/WindowsAzure/Table/ITableSet.cs [hunk 3]
```diff
@@ -165,5 +167,51 @@ public interface ITableSet<TEntity> : IOrderedQueryable<TEntity> where TEntity :
...
+        Task<bool> CreateIfNotExistsAsync();
+
+        /// <summary>
+        ///     Initiates an asynchronous operation to create a table if it does not already exist.        
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="T:System.Threading.CancellationToken"/> to observe while waiting for a task to complete.</param>
+        /// <returns>
+        /// A <see cref="T:System.Threading.Tasks.Task`1"/> object of type <c>bool</c> that represents the asynchronous operation.
+        /// </returns>
+        Task<bool> CreateIfNotExistsAsync(CancellationToken cancellationToken);
```
Comment (dtretyakov) [inline src/WindowsAzure/Table/ITableSet.cs]: Change argument in this method like that: CancellationToken cancellationToken = default (CancellationToken)
And remove all others CreateIfNotExistsAsync method overloads

**Comment by dtretyakov** [comment]
I've a few comments for this PR, please review it. Thanks!

**Comment by tdietrich513** [comment]
Ok, those changes have been completed.

**Pull request merged**